backend/src/main/java/gov/cdc/usds/simplereport/db/model/TestEvent.java

```diff
@@ -68,7 +68,7 @@ public TestEvent(TestOrder testOrder, Boolean hasPriorTests, Set<Result> results
   }
 
   private TestEvent(TestOrder order, Boolean hasPriorTests) {
-    super(order.getPatient(), order.getFacility(), order.getDeviceSpecimen());
```

Comment:
if `order` has a different device type and specimen type than are default at the facility, won't this change cause the constructor to create a test event with the facility defaults instead of the order device and specimen types like it did before?